cfe_internal/enterprise/federation/federation.cf

```diff
@@ -397,12 +399,33 @@ bundle agent superhub_schema
       "$(cfengine_enterprise_federation:config.bin_dir)/psql_wrapper.sh"
         arglist => {
                      "cfdb",
-                     "'select superhub_schema();'",
+                     "\"select superhub_schema('$(sys.key_digest)');\"",
```

Comment:
I personally dislike escaping strings. So, if you also dislike escaping strings, then you can change your outer double quotes to backticks ``` https://docs.cfengine.com/docs/3.12/reference-language-concepts-variables.html#quoting